components/raftstore/src/store/fsm/peer.rs

```diff
@@ -2226,6 +2202,9 @@ where
                 "peer_id" => self.fsm.peer_id(),
                 "split_count" => regions.len(),
             );
+            if need_check_split {
+                self.fsm.peer.schedule_check_split(self.ctx);
```

Comment:
The `estimated_size` may be far from the real size. Should we always call `schedule_check_split` here?

Reply:
If `estimated_size ` is some, we will get approximate size in split-check thread again and decide whether to split.

Reply:
If `approximate_size` is some and the threshold is not reached, there is no split check.